apps/desktop/src/views/SettingsView/components/AiServices/components/ProviderList.vue

```
@@ -39,16 +39,26 @@
+        props.providers.filter(
+            (provider) =>
+                !PROMOTED_PROVIDER_DRIVERS.has(provider.driver) && PRIMARY_NAMES.has(provider.name)
+        )
     );
 
     const otherProviders = computed(() =>
-        props.providers.filter((p) => p.name !== PROMOTED_NAME && !PRIMARY_NAMES.has(p.name))
+        props.providers.filter(
+            (provider) =>
+                !PROMOTED_PROVIDER_DRIVERS.has(provider.driver) &&
+                !PRIMARY_NAMES.has(provider.name)
```

ESLint

[error] Delete `⏎···············`

(prettier/prettier)